tests/bluechi_test/machine.py

```diff
@@ -122,6 +122,22 @@ def copy_container_script(self, script_file_name: str):
              with content:\n{content}")
         self.create_file(target_dir, script_file_name, content)
 
+    def copy_machine_lib(self):
+        LOGGER.info("Copying machine lib...")
+        curr_dir = os.getcwd()
+        source_dir = os.path.join(curr_dir, "..", "..", "..", "bluechi_test", "bluechilib")
+        if not os.path.isdir(source_dir):
+            # This message is relevant for `finish` phase, where code coverage report is being created
+            LOGGER.info("bluechilib directory not found, proceeding")
+            return
+        self.exec_run("mkdir -p /tmp/bluechi_test/bluechilib")
+        for filename in os.listdir(source_dir):
+            source_path = os.path.join(source_dir, filename)
+            if os.path.isfile(source_path):
```

Comment:
```suggestion
            if os.path.isfile(source_path) and source_path.endswith(".py"):
```
I think it doesn't hurt to copy other files as well, but we are interested only in python files.

Reply:
Done